app/assets/stylesheets/provider/_commons.scss

```diff
@@ -99,6 +99,7 @@ ul, p, ol, dl {
   min-height: line-height-times(1);
   padding: line-height-times(1/2) 0 line-height-times(1/2) 40%;
   border-bottom: $border-width solid $border-color;
+  list-style: none;
```

Comment:
Can this change break other views?

Reply:
I haven't found any other `dd` with a list inside and this style is usually set to all lists. I think this won't do any harm.